ci_scripts/pylint/pylintrc

```diff
@@ -236,7 +236,23 @@ disable=all
 # W1402: Anomalous Unicode escape in byte string: \'%s\'. String constant might be missing an r or u prefix.
 # W1501: "%s" is not a valid mode for open.
 # # # enables here # # #
-enable = C0102, C0103, C0111, C0112, C0121, C0202, C0203, C0204, C0301, C0302, C0303, C0304, C0322, C0323, C0324, C0325, C0326, C1001, E0001, E0011, E0012, E0100, E0101, E0102, E0103, E0104, E0105, E0106, E0107, E0108, E0202, E0203, E0211, E0213, E0221, E0222, E0235, E0501, E0502, E0503, E0601, E0602, E0603, E0604, E0611, E0701, E0702, E0710, E0711, E0712, E1001, E1002, E1003, E1004, E1101, E1102, E1103, E1111, E1120, E1121, E1122, E1123, E1124, E1125, E1200, E1201, E1205, E1206, E1300, E1301, E1302, E1303, E1304, E1305, E1306, E1310, F0001, F0002, F0003, F0004, F0010, F0202, F0220, F0321, F0401, I0001, I0010, I0011, I0012, I0014, I0020, I0021, I0022, R0201, R0401, R0801, R0901, R0902, R0904, R0911, R0912, R0913, R0914, R0915, R0921, R0922, R0923, W0101, W0102, W0104, W0105, W0106, W0107, W0108, W0109, W0110, W0120, W0121, W0122, W0141, W0142, W0150, W0199, W0201, W0211, W0212, W0221, W0222, W0223, W0231, W0232, W0233, W0234, W0301, W0311, W0312, W0331, W0332, W0333, W0401, W0402, W0403, W0404, W0406, W0410, W0511, W0512, W0601, W0602, W0603, W0604, W0611, W0612, W0613, W0614, W0621, W0622, W0623, W0631, W0632, W0633, W0701, W0702, W0703, W0704, W0710, W0711, W0712, W1001, W1111, W1201, W1300, W1301, W1401, W1402, W1501, W9000
+enable = C0102, C0103, C0111, C0112, C0121, C0202, C0203, C0204, C0301, C0302,
```

Comment:
When we first made pylint, I recall asking @ruyimarone to put each of these in their own lines and a comment per line when an explanation of what it did. He said it doesn't work and that all the codes needed to be in the same line. Is that not the case anymore? (perhaps because we're using py2?).

If multi-lines work, can we move the docs to the side of each code?